src/Nethermind/Chains/chiado.json

```diff
@@ -69,6 +68,12 @@
     "eip3855TransitionTimestamp": "0x646e0e4c",
     "eip3860TransitionTimestamp": "0x646e0e4c",
     "eip4895TransitionTimestamp": "0x646e0e4c",
+    "eip1153TransitionTimestamp": "0x65ba8e4c",
+    "eip4788TransitionTimestamp": "0x65ba8e4c",
+    "eip4844TransitionTimestamp": "0x65ba8e4c",
+    "eip5656TransitionTimestamp": "0x65ba8e4c",
+    "eip6780TransitionTimestamp": "0x65ba8e4c",
+    "eip4788ContractAddress": "0x000F3df6D732807Ef1319fB7B8bB8522d0Beac02",
```

Comment:
4788 address doesn't have to be specified

Reply:
Then why do we have it specified for Goerli?